frontend/libs/studio-components/src/components/StudioCodeFragment/StudioCodeFragment.module.css

```diff
@@ -1,8 +1,12 @@
 .code {
-  background-color: var(--fds-semantic-surface-neutral-subtle);
+  flex: 1;
+  background-color: rgb(221, 221, 234);
   border: 1px solid var(--fds-semantic-border-neutral-subtle);
   border-radius: var(--fds-border_radius-small);
   font-family: 'Courier New', monospace;
-  font-size: 0.8em;
+  font-size: 0.9em;
   padding: 0 var(--fds-spacing-1);
+  overflow: auto;
+  height: 28px;
```

Comment:
If we do not have a suited ds-variable we should us `em` or `rem` since these are more dynamic measures than px 😊 Maybe `2em`?

Reply:
From my perspective, we should avoid hard-coding a height for the component within studio-components. We might need to use this component elsewhere in the codebase where the content requires a height greater than 28px. Studio Components should be flexible, so I suggest not specifying either overflow or height here. Instead, we should pass a className into the component to handle any specific needs based on where it is used.